### ISSUE TITLE
pytest 5.1.1 does not support python 3.7.4

### ISSUE THREAD

**Issue by acerv** [pytest-dev/pytest#5789]
Hi,

I'm trying to use pytest 5.1.1 with python 3.7.4, but I obtain the following error:
```
(.venv374_test) C:\userdata>pytest --help
Traceback (most recent call last):
  File "C:\userdata\Python37-32\Lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\userdata\Python37-32\Lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\userdata\.venv374_test\Scripts\pytest.exe\__main__.py", line 9, in <module>
  File "c:\userdata\.venv374_test\lib\site-packages\_pytest\config\__init__.py", line 58, in main
    config = _prepareconfig(args, plugins)
  File "c:\userdata\.venv374_test\lib\site-packages\_pytest\config\__init__.py", line 208, in _prepareconfig
    pluginmanager=pluginmanager, args=args
  File "c:\userdata\.venv374_test\lib\site-packages\pluggy\hooks.py", line 289, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "c:\userdata\.venv374_test\lib\site-packages\pluggy\manager.py", line 87, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "c:\userdata\.venv374_test\lib\site-packages\pluggy\manager.py", line 81, in <lambda>
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
  File "c:\userdata\.venv374_test\lib\site-packages\pluggy\callers.py", line 203, in _multicall
    gen.send(outcome)
  File "c:\userdata\.venv374_test\lib\site-packages\_pytest\helpconfig.py", line 89, in pytest_cmdline_parse
    config = outcome.get_result()
  File "c:\userdata\.venv374_test\lib\site-packages\pluggy\callers.py", line 80, in get_result
    raise ex[1].with_traceback(ex[2])
  File "c:\userdata\.venv374_test\lib\site-packages\pluggy\callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "c:\userdata\.venv374_test\lib\site-packages\_pytest\config\__init__.py", line 716, in pytest_cmdline_parse
    self.parse(args)
  File "c:\userdata\.venv374_test\lib\site-packages\_pytest\config\__init__.py", line 924, in parse
    self._preparse(args, addopts=addopts)
  File "c:\userdata\.venv374_test\lib\site-packages\_pytest\config\__init__.py", line 880, in _preparse
    early_config=self, args=args, parser=self._parser
  File "c:\userdata\.venv374_test\lib\site-packages\pluggy\hooks.py", line 289, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "c:\userdata\.venv374_test\lib\site-packages\pluggy\manager.py", line 87, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "c:\userdata\.venv374_test\lib\site-packages\pluggy\manager.py", line 81, in <lambda>
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
  File "c:\userdata\.venv374_test\lib\site-packages\pluggy\callers.py", line 208, in _multicall
    return outcome.get_result()
  File "c:\userdata\.venv374_test\lib\site-packages\pluggy\callers.py", line 80, in get_result
    raise ex[1].with_traceback(ex[2])
  File "c:\userdata\.venv374_test\lib\site-packages\pluggy\callers.py", line 182, in _multicall
    next(gen)  # first yield
  File "c:\userdata\.venv374_test\lib\site-packages\_pytest\capture.py", line 42, in pytest_load_initial_conftests
    _py36_windowsconsoleio_workaround(sys.stdout)
  File "c:\userdata\.venv374_test\lib\site-packages\_pytest\capture.py", line 819, in _py36_windowsconsoleio_workaround
    sys.stdin = _reopen_stdio(sys.stdin, "rb")
  File "c:\userdata\.venv374_test\lib\site-packages\_pytest\capture.py", line 812, in _reopen_stdio
    open(os.dup(f.fileno()), mode, buffering),
OSError: [WinError 87] The parameter is incorrect
```

According with the unittests, it looks like Travis CI is running tests on python 3.7.1, but python 3.7.4 is the last stable version available. Is this is the maximum required version to work with pytest?

Thanks,
Andrea

**Comment by The-Compiler** [comment]
It's a bug in Python 3.7.4 with Windows 7 - it'll be fixed in Python 3.7.5, until then you might want to use 3.7.3. See #5724 for details.

**Comment by acerv** [comment]
Thank you very much
